airbyte-cron/build.gradle

```diff
@@ -31,6 +31,19 @@ application {
     applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0']
 }
 
+Properties env = new Properties()
+rootProject.file('.env.dev').withInputStream { env.load(it) }
```

Comment:
Why are we getting `.env.dev` and not `.env`?

Reply:
@gosusnp The theory is that this is being used for quick, local testing and therefore use the dev version.  However, if the .env.dev is not reliable, etc, this could easily be switched.